src/fem/dh_cell_accessor.hh

```diff
@@ -0,0 +1,178 @@
+/*!
+ *
+﻿ * Copyright (C) 2015 Technical University of Liberec.  All rights reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License version 3 as published by the
+ * Free Software Foundation. (http://www.gnu.org/licenses/gpl-3.0.en.html)
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+ *
+ *
+ * @file    dh_cell_accessor.hh
+ * @brief
+ * @author  David Flanderka
+ */
+
+#ifndef DH_CELL_ACCESSOR_HH_
+#define DH_CELL_ACCESSOR_HH_
+
+#include "mesh/accessors.hh"
+#include "fem/dofhandler.hh"
+
+/**
+ * Cell accessor allow iterate over DOF handler cells.
+ *
+ * Iterating is possible over different ranges of local and ghost elements.
+ */
+class DHCellAccessor {
```

Comment:
- Remove dereference operator.
- 'elm' method instead of 'element_accessor'
- add dim() method 
- add fe method (similar to DoFHanlderMultidim.fe(...))